File: fib.cs
// Calculate the n^th Fibonacci number.
class fib
{
  public static int Calcfib(int n)
  {
    if (n < 2)
    {
      return n;
    }
    else
    {
      return Calcfib(n - 1) + Calcfib(n - 2);
    }
  }

  public static void Main()
  {
    int res = Calcfib(20);
    System.Console.WriteLine(res);
  }

}

File: fib.pas
{Calculate the n^th Fibonacci number.}

program rfib;

function fib(n: integer): integer;
begin
  if n<2 then
    fib := n
  else
    fib := fib(n-1) + fib(n-2);
end; 

begin
  writeLn(fib(20));
end.
